client/src/pages/mypage/withdraw/WithdrawPage.tsx

```diff
@@ -15,12 +15,15 @@ const WithdrawPage = () => {
   return (
     <MainLayout backgroundColor="white">
       <TopNav>
-        <TopNav.Item displayName="뒤로가기" noEmphasis routePath="-1" />
+        {step !== 'withdrawalCompleted' ? (
+          <TopNav.Item displayName={'뒤로가기'} noEmphasis routePath="-1" />
+        ) : undefined}
```

Comment:
```suggestion
        {step !== 'withdrawalCompleted' && <TopNav.Item displayName={'뒤로가기'} noEmphasis routePath="-1" />}
```

아래와 비슷하게 undefined를 사용하지 않고 이렇게 쓸 수 있을 것 같아요

Reply:
오! 훨씬 깔끔하고 좋네요! 이 방식으로 바꿔둘게요 ㅎㅎ